src/Sentry.Unity/Integrations/UnityErrorLogException.cs

```diff
@@ -0,0 +1,170 @@
+            frames.Reverse();
+
+            var stacktrace = new SentryStackTrace { Frames = frames };
+
+            return new SentryException
+            {
+                Stacktrace = stacktrace,
+                Type = ExceptionType,
+                Value = _logString,
+                Mechanism = new Mechanism
+                {
+                    Handled = true,
+                    Type = "unity.log"
+                }
+            };
+        }
+
+        private const string AtFileMarker = " (at ";
+
+        private List<SentryStackFrame> ParseStackTrace(string stackTrace)
+        {
+            // Example: Sentry.Unity.Integrations.UnityLogHandlerIntegration:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[]) (at UnityLogHandlerIntegration.cs:89)
+            // This follows the following format:
+            // Module.Class.Method[.Invoke] (arguments) (at filepath:linenumber)
+            // The ':linenumber' is optional and will be omitted in builds
+
+            var frames = new List<SentryStackFrame>();
+            var stackList = stackTrace.Split('\n');
+
+            foreach (var line in stackList)
+            {
+                var item = line.TrimEnd('\r');
+                if (string.IsNullOrEmpty(item))
+                {
+                    continue;
+                }
+
+                var frame = ParseStackFrame(item);
+                if (_options is not null)
+                {
+                    frame.ConfigureAppFrame(_options);
+                }
+                frames.Add(frame);
+            }
+
+            return frames;
+        }
+
+        private static SentryStackFrame ParseStackFrame(string stackFrameLine)
+        {
+            var closingParenthesis = stackFrameLine.IndexOf(')');
+            if (closingParenthesis == -1)
+            {
+                return CreateBasicStackFrame(stackFrameLine);
+            }
+
+            try
+            {
+                var functionName = stackFrameLine.Substring(0, closingParenthesis + 1);
+                var remainingText = stackFrameLine.Substring(closingParenthesis + 1);
+
+                if (!remainingText.StartsWith(AtFileMarker))
+                {
+                    // If it does not start with '(at' it's an unknown format. We're falling back to a basic stackframe
+                    return CreateBasicStackFrame(stackFrameLine);
+                }
+
+                var (filename, lineNo) = ParseFileLocation(remainingText);
+                var filenameWithoutZeroes = StripZeroes(filename);
+
+                return new SentryStackFrame
+                {
+                    FileName = TryResolveFileNameForMono(filenameWithoutZeroes),
+                    AbsolutePath = filenameWithoutZeroes,
+                    Function = functionName,
+                    LineNumber = lineNo == -1 ? null : lineNo
+                };
+            }
+            catch (Exception)
```

Comment:
concerning that we catch all and don't have any indication of what went wrong. debug logging at least?